public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <title>ikinyarwanda-frontend</title>
```

Comment:
@ceelogre I think, for now, you can change this title to "**Ikinyarwanda**". Because end-users do not neet know if the web is "frontend"